ui/public/checkBrowser.js

```diff
@@ -0,0 +1,27 @@
+function checkBrowser() {
+  if (!window.__supported_browsers__.test(navigator.userAgent)) {
+    var text
+    if (navigator.language.indexOf('zh') === 0) {
+      text =
+        '您的浏览器版本已过期，推荐使用最新版本的 Chrome 或 Edge 浏览器以便获得更好的体验。'
+    } else {
+      text =
+        'Your browser version is outdated, recommend to use the latest Chrome or Edge to get better experience.'
```

Comment:
Sometimes the page is simply broken, prefer to avoid the soft "recommend"

Reply:
We also support Firefox

Reply:
Ok, how about "latest modern browsers"?

Reply:
Modern browser is a word for developers and users don't know what it means precisely. I prefer to explicitly list out our supported browsers